src/topology/constructions.lean

```diff
@@ -111,6 +111,40 @@ nhds_induced coe a
 
 end topα
 
+def cofinite_topology (α : Type*) : topological_space α :=
```

Comment:
The linter wants a docstring here.

Reply:
I'm sorry about that, I had the impression that CI was already happy with linting on the previous try.

Reply:
No worries, I just posted the comment to hopefully save you a click or two. By the way, if you add the "awaiting-CI" label to a PR that is currently running CI, it should be automatically removed if everything passes.